documentation/usage/settings.md

```diff
@@ -2,8 +2,101 @@
 sidebar_position: 1
 ---
 
+import CaptionedImage from "/src/components/CaptionedImage";
+
 # Game Settings
 
+In addition to the settings available in the original Jak games, OpenGOAL provides some additional settings that will allow you to tweak and improve the games in new ways. This page will highlight some of these new settings and provide examples for them, but there are still more to find in game. Once you've had a look through this page, [try changing them yourself in game](/docs/usage/installation)!
+
+## Subtitles (and languages)
+
+**Found in Options > Game Options**
+
+One major new addition to OpenGOAL that was not present in the original game are the subtitles. These were not part of the original game, but instead are a community effort to transcribe and translate the game into subtitles for different languages, making it easier for more people to enjoy.
+
+Additionally all of the original available audio and text options for the game are available as settings, regardless of what version is on your disc!
```

Comment:
...original ~available~ audio...

Reply:
Additionally I would specify the six original languages - black label doesn't show Japanese, so saying 'the original options' would be incorrect, for example.

Reply:
Cheers, changing that second sentence to:

`Additionally all six original audio and text languages for the game are available as settings, regardless of what version is on your disc!`

My thinking with specifying the original options wasn't about them being options on every disc so much as clarifying those are community created, but I think saying the six original languages covers that nicely. Hopefully this sentence reads better!